### ISSUE TITLE
Form Daily2- Temperature

### ISSUE THREAD

**Issue by shadrackkibet** [africanmathsinitiative/Climsoft#57]
Lesotho Met data entry team would like to have a check for Tmin and Tmax being done on the form. There is no instance whereby Tmax < Tmin and therefore they would like Climsoft not to allow that.

**Comment by Patowhiz** [comment]
@mhabimana and @smachua what's your opinion on this. This can be done but the assumption here will be that one of the two, either tmin or tmax must be entered before the other for such a comparison to be valid

**Comment by mhabimana** [comment]
@Patowhiz : The reason why there is no check for TMax and TMin on the form Daily2 is because,this form is meant to enter one element at time. If this was meant for many elements at a time,this could be obvious. In climsoft,this check is performed during the QC check (enter-element comparison check).

**Comment by Patowhiz** [comment]
@mhabimana I agree with you. We could leave this as it is. In addition, @shadrackkibet gave me logical reason as to why the lesotho teams preferred also to have the check on the form. And this can be trivially done on the data entry forms by just having them do the comparison during entry of these 2 elements only. Then we could just show a prompt(e.g Message box) about it and let the user to proceed with data entry, knowingly that he/she made an error. 
I find it easier and quicker from the users perspective to check on errors in real time rather than in batch(which QC check does).

**Comment by mhabimana** [comment]
@Patowhiz : I suggest,you can try to implement this check and then we can test how it works.My view.
